### PR TITLE
Audit report

### PR DESCRIPTION
**Pull request by rmeissner**

### CODE REVIEW

**Review by fedgiac — APPROVED** [review]
:heavy_check_mark:

**Review by auryn-macmillan — APPROVED** [review]
Looks good to me.

File: README.md [hunk 1]
```diff
@@ -91,10 +91,15 @@ The contracts have been developed with [Solidity 0.8.0](https://github.com/ether
 - Resolve oracle (e.g. answer question on Rinkeby https://reality.eth.link/app/)
 - `yarn hardhat --network rinkeby executeProposal --module <module_address> --question <question_id_from_realitio> --proposal-file sample_proposal.json`
 
-### Audits
+### Audit Notes
```
Comment (rmeissner) [inline README.md]: Thus should not be changes @auryn-macmillan :D

**Review by auryn-macmillan — APPROVED** [review]
Looks good to me.

**Pull request merged**